src/styles.css

```diff
@@ -0,0 +1,667 @@
+body {
+	background-color: white;
+	color: #565656;
+	width: 100%;
+    font-size: 15px;
```

Comment:
not consistent indent across all files. Always try to follow the same code format rules